explorer/websocket.go

```diff
@@ -180,10 +180,7 @@ func (wsh *WebsocketHub) pingClients() chan<- struct{} {
 			select {
 			case <-ticker.C:
 				wsh.HubRelay <- pstypes.HubMessage{Signal: sigPingAndUserCount}
-			case _, ok := <-stopPing:
-				if !ok {
```

Comment:
This should have been `if ok` to check if it was a send rather than a close, but this entire check was just for debugging.  No one is sending on `stopPing`